repository/repository.go

```diff
@@ -0,0 +1,204 @@
+package repository
+
+import (
+	"database/sql"
+	"go_final_project/structurs"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	_ "modernc.org/sqlite"
+)
+
+type DB struct {
+	db *sql.DB
+}
+
+func NewDBwork(db *sql.DB) DB {
+	return DB{db: db}
+}
+func (s DB) AddDB(DBFile string) error {
+	_, err := os.Create(DBFile)
+	if err != nil {
+		return err
+	}
+	_, err = s.db.Exec("CREATE TABLE IF NOT EXISTS scheduler (id INTEGER PRIMARY KEY AUTOINCREMENT, date TEXT NOT NULL CHECK(date !=''), title TEXT NOT NULL CHECK(title !=''), comment TEXT, repeat TEXT);")
+	if err != nil {
+		return err
+	}
+	_, err = s.db.Exec("CREATE INDEX ID_Date ON scheduler (date);")
+	if err != nil {
+		return err
+	}
+	return nil
+}
+func (s DB) CheckTable(DBFile string) error {
```

Comment:
Не используемая переменная

```suggestion
func (s DB) CheckTable() error {
```